example/ida/ctype_propagation.py

```diff
@@ -255,7 +255,8 @@ def analyse_function():
         done.add((lbl, state))
         symbexec_engine = TypePropagationEngine(ir_arch, types_mngr, state)
 
-        assert lbl in ir_arch.blocks
+        if lbl not in ir_arch.blocks:
```

Comment:
If you're going to continue (and skip the rest of the loop), there is no need to instance a new symbexec engine (ie, the `if` statement should be upper)